include/modem/lte_lc.h

```diff
@@ -479,6 +631,27 @@ int lte_lc_func_mode_get(enum lte_lc_func_mode *mode);
  */
 int lte_lc_lte_mode_get(enum lte_lc_lte_mode *mode);
 
+/**@brief Get connection evaluation parameters. Connection evaluation parameters can be used to
+ *	  determine the energy efficiency of data transmission prior to the actual
+ *	  data transmission.
+ *
+ * @param params Pointer to structure to hold connection evaluation parameters.
+ *
+ * @return Zero on success or (negative) error code otherwise.
```

Comment:
With the comments below explaining that return value can be a positive integer, I think this comment is a bit off.